app/models/audit.rb

```diff
@@ -94,6 +94,10 @@ def completed?
     observations.audit_activities_completed.any?
   end
 
+  def tasks_completed_on
+    observations.audit_activities_completed.last.at
```

Comment:
I assume there's only ever one of these so taking the last is fine, rather than sorting by `at`.